src/main/java/com/almightyalpaca/discord/jdabutler/commands/commands/SlowmodeCommand.java

```diff
@@ -0,0 +1,56 @@
+package com.almightyalpaca.discord.jdabutler.commands.commands;
+
+import com.almightyalpaca.discord.jdabutler.Bot;
+import com.almightyalpaca.discord.jdabutler.commands.Command;
+import net.dv8tion.jda.core.entities.Message;
+import net.dv8tion.jda.core.entities.TextChannel;
+import net.dv8tion.jda.core.entities.User;
+import net.dv8tion.jda.core.events.message.guild.GuildMessageReceivedEvent;
+
+public class SlowmodeCommand extends Command
+{
+    @Override
+    public void dispatch(final User sender, final TextChannel channel, final Message message, final String content, final GuildMessageReceivedEvent event)
+    {
+        if (!Bot.isHelper(sender))
+        {
+            this.sendFailed(message);
+            return;
+        }
+
+        int seconds;
+
+        String args = content.trim().toLowerCase();
+
+        if (args.equals("off") || args.equals("false"))
```

Comment:
What about empty args? Currently it will just try to parse "", fail and reply with "Could not parse argument"